yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -156,87 +185,68 @@ func packAndSendAssessMigrationPayload(status string, errMsg string) {
 		}),
 	}
 
-	unsupportedDatatypesList := lo.Map(assessmentReport.UnsupportedDataTypes, func(datatype utils.TableColumnsDataTypes, _ int) string {
-		return datatype.DataType
-	})
+	explanation, err := buildMigrationComplexityExplanation(source.DBType, assessmentReport, "")
+	if err != nil {
+		log.Errorf("failed to build migration complexity explanation: %v", err)
+	}
+
+	var obfuscatedIssues []callhome.AssessmentIssueCallhome
+	for _, issue := range assessmentReport.Issues {
+		obfuscatedIssue := callhome.AssessmentIssueCallhome{
+			Category:              issue.Category,
+			CategoryDescription:   issue.CategoryDescription,
+			Type:                  issue.Type,
+			Name:                  issue.Name,
+			Description:           issue.Description,
+			Impact:                issue.Impact,
+			ObjectType:            issue.ObjectType,
+			ObjectName:            constants.OBFUSCATE_STRING,
+			SqlStatement:          constants.OBFUSCATE_STRING, // TODO(future): we can obfuscate sensitive info in SQL statement
```

Comment:
I am not sure if that would be easily possible to obfuscate the SQL statement and if there is any need to do that.

Reply:
will remove ObjectName and SqlStatement field